pokemongo_bot/__init__.py

```diff
@@ -160,10 +112,50 @@ def _setup_api(self):
         print('[#] Stardust: {}'.format(stardust))
         print('[#] Pokecoins: {}'.format(pokecoins))
         self.get_player_info()
-        print('[#]')
 
+        if self.config.firsttrans:
+            self.first_transfer()
+            
+        print('[#]')
         self.update_inventory();
+        
+
+    def first_transfer(self):
+        pokemon_stock = {}
+        self.api.get_player().get_inventory()
+        inventory_req = self.api.call()
+        inventory_dict = inventory_req['responses']['GET_INVENTORY']['inventory_delta']['inventory_items']
+       
+        print('[x] First Transfer.')
+        
+        for pokemon in inventory_dict:
+            try:
+                id1 = pokemon['inventory_item_data']['pokemon_data']['pokemon_id']
+                id2 = pokemon['inventory_item_data']['pokemon_data']['id']
+                id3 = pokemon['inventory_item_data']['pokemon_data']['cp']
+
```

Comment:
consider renaming id{1,2,3} to something more meaning, like `pokemon_id`, `id`, and `cp`